roboprop_client/views.py

```diff
@@ -544,3 +544,42 @@ def add_metadata(request, name):
     else:
         messages.error(request, "No metadata available")
         return redirect("mymodels")
+
+
+def update_models_from_blendkit(request):
+    if request.method == "POST":
+        # get index.json
+        index = _check_and_get_index(request)
+        for model in index:
+            # check if model has key assetBaseId, i.e is from blendkit
+            if "assetBaseId" in index[model]:
```

Comment:
Good as it is now, but maybe we will need a `source` field in the future?

Reply:
`source` as in what library it originally came from? (own, fuel, blendkit etc etc.). Yes I would agree we most likely should have it in future. thank you